src/state/controls/Control.spec.ts

```diff
@@ -43,4 +48,19 @@ describe('control', () => {
         });
         stub.restore();
     });
+
+    it('lets you update cost', () => {
+        const buttonDiff = {
+            cost: 200,
+        };
+        const updatedButton = Object.assign({}, buttonData2, buttonDiff);
+        const stub = sinon.stub(mockClient, 'updateControls');
+        control.setCost(200);
```

Comment:
Tests should be easy to change so we tend to avoid hard coded values. Here you could do `setCost(buttonDiff.cost)` and that way there's no bare values.